docs/languages/javascript.md

```diff
@@ -247,6 +247,30 @@ VS Code automatically suggests some common code simplifications such as converti
 
 Set `"javascript.suggestionActions.enabled"` to `false` to disable suggestions.
 
+## Enhancing completions with AI
+
+[GitHub Copilot](https://copilot.github.com/) is an AI-powered code completion tool that helps you write code faster and smarter. You can use the [Copilot extension](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) in VS Code to generate code, or to learn from the code it generates.
+
+![Copilot extension in marketplace](./images/artifical-intelligence/copilot-extension.png)
+
+GitHub Copilot provides suggestions for numerous languages and a wide variety of frameworks, and it works especially well for Python, JavaScript, TypeScript, Ruby, Go, C# and C++.
+
+You can learn more about how to get started with Copilot in the [Copilot documentation](../editor/artificial-intelligence.md).
+
+Once you have the Copilot extension installed and enabled, you can test it our for your JavaScript projects.
+
+Create a new file - you can use the **File: New File** command in the Command Palette (`kbstyle(F1)`).
+
+In the JavaScript file, type the following function header:
+
+```js
+function calculateDaysBetweenDates(begin, end) {
+```
+
+Copilot will provide a suggestion like the following - use `Tab` to accept the suggestion:
+
+![JavaScript ghost text suggestion](../editor/images/artifical-intelligence/js-suggest.png)
```

Comment:
Prefer to create a new image than reach into another subdirectory. This makes it easier to move articles and their corresponding images (and not accidently remove an image used by another article).

Reply:
Got it! Recreated this image in all the docs areas where I reference it, and switched to full paths.